package.json

```diff
@@ -114,9 +117,10 @@
     "ci:definitionsCheck": "node ./ci/definitionsCheck.js",
     "definitions": "node ./resources/buildConfigDefinitions.js && prettier --write 'src/Options/*.js'",
     "docs": "jsdoc -c ./jsdoc-conf.json",
-    "lint": "flow && eslint --cache ./",
+    "lint": "eslint --cache ./",
     "lint-fix": "eslint --fix --cache ./",
-    "build": "babel src/ -d lib/ --copy-files",
+    "build": "babel src/ -d lib/ --copy-files --extensions '.ts,.js'",
+    "build:types": "tsc && prettier --write 'types/{**/*,*}.ts' && npm run lint-fix",
```

Comment:
Could you please add a `test:types` script and a CI job for it, like we have in the JS SDK? I think you can just copy both from https://github.com/parse-community/Parse-SDK-JS/pull/1985/files